components/binance/browser/binance_service.cc

```diff
@@ -256,6 +256,14 @@ bool BinanceService::SetAccessTokens(const std::string& access_token,
   access_token_ = access_token;
   refresh_token_ = refresh_token;
 
+  PrefService* prefs = user_prefs::UserPrefs::Get(context_);
+
+  if (access_token.empty() && refresh_token.empty()) {
```

Comment:
Maybe split this path out into a `ResetAccessTokens` function for extra clarity?

Reply:
good suggestion, updated